test/Source/DoctrineORMQuerySourceIteratorTest.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Exporter\Test\Source;
+
+use Doctrine\ORM\Configuration;
+use Doctrine\ORM\EntityManagerInterface;
+use Doctrine\ORM\Query;
+use Exporter\Test\Source\Fixtures\DoctrineORMQuerySourceIterator;
+use Exporter\Test\Source\Fixtures\ObjectWithToString;
+
+/**
+ * @author Joseph Maarek <josephmaarek@gmail.com>
+ */
+class DoctrineORMQuerySourceIteratorTest extends \PHPUnit_Framework_TestCase
```

Comment:
`final`

Reply:
Not needed for tests

Reply:
If a class can be final, it should be final by default.